packaging/installer/methods/kubernetes.md

```diff
@@ -43,147 +33,138 @@ helm repo add netdata https://netdata.github.io/helmchart/
+Cloud](https://app.netdata.cloud), you must first [claim](/claim/README.md) your Kubernetes cluster. Claiming securely
+connects your Kubernetes cluster to stream metrics data to Netdata Cloud, enabling Kubernetes-specific visualizations
+like the health map and time-series composite charts.
 
-```bash
-helm install --set a.b.c=xyz netdata netdata/netdata
-helm upgrade --set a.b.c=xyz netdata netdata/netdata
-```
+First, find your claiming script in Netdata Cloud by clicking on your Space's dropdown, then **Manage your Space**.
+Click the **Nodes** tab to reveal the `netdata-claim.sh` script for your Space in Netdata Cloud. You need the `TOKEN`
+and `ROOM` values.
 
-For example, to change the size of the persistent metrics volume on the parent node:
+Next, create a file called `override.yml`.
 
 ```bash
-helm install --set parent.database.volumesize=4Gi netdata netdata/netdata
-helm upgrade --set parent.database.volumesize=4Gi netdata netdata/netdata
+touch override.yml
 ```
 
-### Configure service discovery
-
-As mentioned in the introduction, Netdata has a [service discovery
-plugin](https://github.com/netdata/agent-service-discovery/#service-discovery) to identify compatible pods and collect
-metrics from the service they run. The Netdata Helm chart installs this service discovery plugin into your k8s cluster.
-
-Service discovery scans your cluster for pods exposed on certain ports and with certain image names. By default, it
-looks for its supported services on the ports they most commonly listen on, and using default image names. Service
-discovery currently supports [popular
-applications](https://github.com/netdata/helmchart#service-discovery-and-supported-services), plus any endpoints covered
-by our [generic Prometheus collector](https://learn.netdata.cloud/docs/agent/collectors/go.d.plugin/modules/prometheus).
-
-If you haven't changed listening ports, image names, or other defaults, service discovery should find your pods, create
-the proper configurations based on the service that pod runs, and begin monitoring them immediately after deployment.
-
-However, if you have changed some of these defaults, you need to copy a file from the Netdata Helm chart repository,
-make your edits, and pass the changed file to `helm install`/`helm upgrade`.
+Paste the following into your `override.yml` file, replacing instances of `ROOM` and `TOKEN` with those from the
+claiming script from Netdata Cloud. These settings claim your `parent`/`child` nodes to Netdata Cloud and store more
+metrics in the nodes' time-series databases.
+
+```yaml
+parent:
+  claiming:
+    enabled: true
+    token: "TOKEN"
+    rooms: "ROOM"
+  configs:
+    netdata:
+      data: |
+        [global]
+          memory mode = dbengine
+          dbengine_multihost_disk_space = 4000
+          default memory mode = dbengine
+          
+        [plugins]
+          apps = yes
+          proc = yes
+          go.d = yes
+          python.d = yes
+ 
```

Comment:
why do we want to have data collection plugins enabled on the parent instance?

Reply:
Good question! I _just_ realized that one of the child pods runs on the same node as the parent, and would have those same plugins enabled. So yes, it's probably best to remove those lines. I'll do that.